system/Autoloader/Autoloader.php

```diff
@@ -280,11 +280,14 @@ protected function loadInNamespace(string $class)
         }
 
         foreach ($this->prefixes as $namespace => $directories) {
+            $classNamespaceFind = strpos($class, $namespace);
+            $fileNamespace      = str_replace('\\', DIRECTORY_SEPARATOR, substr($class, strlen($namespace)));
```

Comment:
Thank you for refactoring. The intent is good. But the variables are a bit not good.

If you see `$classNamespaceFind` or `$fileNamespace`, what do you think the variable represents?

I guess `$classNamespaceFind` is true when the class namespace is found.
Also I guess `$fileNamespace` means the namespace of the file.